contracts/vaults/Claim.sol

```diff
@@ -33,43 +33,52 @@ contract Claim is Base {
         string calldata _descriptionHash)
     external
     onlyCommittee(_pid)
-    noSubmittedClaims(_pid) {
+    noSubmittedClaims(_pid)
+    {
         require(_beneficiary != address(0), "HVE04");
         // require we are in safetyPeriod
         // solhint-disable-next-line not-rely-on-time
         require(block.timestamp % (generalParameters.withdrawPeriod + generalParameters.safetyPeriod) >=
         generalParameters.withdrawPeriod, "HVE05");
         require(_bountyPercentage <= bountyInfos[_pid].maxBounty, "HVE06");
-
-        submittedClaims[_pid] = SubmittedClaim({
+        uint256 claimId = uint256(keccak256(abi.encodePacked(_pid, block.number)));
```

Comment:
wondering if should check that its not 0? (and then do uint256(keccak256(abi.encodePacked(_pid+1, block.number)) or something like that)
and also to check if exists already?